content/docs/security/ratelimit/global.md

```diff
@@ -0,0 +1,445 @@
+
+### Response headers
+
+When rate limiting is enabled, kgateway adds the following headers to responses. These headers help clients understand their current rate limit status and adapt their behavior accordingly.
+
+| Header | Description | Example |
+|--------|-------------|---------|
+| x-ratelimit-limit | The rate limit ceiling for the given request | `10, 10;w=60` (10 requests per 60 seconds) |
+| x-ratelimit-remaining | The number of requests left for the time window | `5` (5 requests remaining) |
+| x-ratelimit-reset | The time in seconds until the rate limit resets | `30` (rate limit resets in 30 seconds) |
+| x-envoy-ratelimited | Present when the request is rate limited | `true` |
+
+## Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+
+## Step 1: Deploy your Rate Limit Service {#byo-rate-limit-service}
+
+You can bring your own rate limit service that implements the Envoy Rate Limit gRPC protocol. 
+
+To get started, you can try out a demo rate limit service from the kgateway project. For more information, see the [GitHub repo](https://github.com/kgateway-dev/kgateway/tree/main/test/kubernetes/e2e/features/rate_limit/testdata).
+
+1. Create the `kgateway-test` namespace.
+
+   ```sh
+   kubectl create namespace kgateway-test
+   ```
+
+2. Deploy the rate limit service.
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/kgateway-dev/kgateway/refs/heads/main/test/kubernetes/e2e/features/rate_limit/testdata/rate-limit-server.yaml
+   ```
+
+## Step 2: Define the rate limits {#rate-limits}
+
+Define the actual rate limit values (requests per unit time) in your Rate Limit Service. For example, using the [Envoy Rate Limit](https://github.com/envoyproxy/ratelimit) service, you configure the rate limits in its configuration file.
+
+The kgateway example that you deployed in the previous step includes the following rate limit configuration as a Kubernetes ConfigMap.
+
+```sh
+kubectl describe configmap ratelimit-config -n kgateway-test
+```
+
+{{< reuse "docs/snippets/review-table.md" >}}
+
+| Field | Description |
+|-------|-------------|
+| domain | Required. A globally unique identifier to group together a set of rate limit rules. This way, different teams can have their own set of rate limits that don't conflict with each other. |
+| descriptors | A list of key-value pairs that the Rate Limit Service uses to select which rate limit to use on matching requests. Descriptors are case-sensitive. |
+| key | Required. The name for the descriptor to use when matching requests. Later, you use the descriptor key in the kgateway TrafficPolicy to decide which rate limits to apply to requests. The TrafficPolicy expects one of the following values for the descriptor key: `remote_address` for a client IP address, `path` for path matching, a name for header matching, or a custom name for generic matching. |
```

Comment:
should this say: 

The Rate Limit Service expects one of the following .... 

instead of **The TrafficPolicy**?